features/full_tests/native_crash_handling.feature

```diff
@@ -105,80 +105,6 @@ Feature: Native crash reporting
         And the event "severity" equals "error"
         And the event "unhandled" is true
 
-    Scenario: Raise SIGILL
```

Comment:
There are 142 scenarios running on next and only 141 here.  It look like this one has been dropped - was that intentional?

Reply:
Nice catch - that was a scenario I missed. This has now been reinstated in `native_signal_raise.feature`.